wallet/src/ui/app/components/nft-display/index.tsx

```diff
@@ -30,10 +30,17 @@ function NFTDisplayCard({
         useNFTBasicData(nftobj);
 
     const name = nftFields?.name || nftFields?.metadata?.fields?.name;
+    const objIDShort = useMiddleEllipsis(nftObjectID);
+    const nftType = useMiddleEllipsis(
+        ('type' in nftobj.data && nftobj.data.type) || null,
+        20,
```

Comment:
can this be a const value

Reply:
Oh you mean to use constants instead of `20` & `3`? Yeah I will change it (actually I can move the `type` 'extraction' to the `useNFTBasicData` hook as well!)

Reply:
Done